### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by eduvinagre**
- [DEMO LINK](https://eduvinagre.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://eduvinagre.github.io/layout_moyo-header/report/html_report/)

- [X] Header height is set in 1 place (for the links)
- [X] Content is vertically centered (for any header height)
- [X] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [X] Logo is an image wrapped with a link
- [X] **CSS Variable** is used for a blue color
- [X] Pseudo-element is used for a blue line below the active link
- [X] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -15,8 +15,108 @@
...
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav__list">
```
Comment (jv-aquino) [inline src/index.html:45]: You are using the BEM nesting in a wrong way here, try using Mix Block or just the normal nesting conventions

https://en.bem.info/methodology/quick-start/#nesting-1

File: src/index.html [hunk 2]
```diff
@@ -15,8 +15,108 @@
...
+          <li class="header__nav__item-1">
```
Comment (jv-aquino) [inline src/index.html:46]: You didn't need to add numbers for each of them, they have (except for the first one) the same attributes

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,103 @@
...
+
+.header__nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.header__nav__item-1 {
+  padding-right: 20px;
```
Comment (jv-aquino) [inline src/style.css:37]: With this use of classes your code is more prone to errors and difficult to change, so group them more

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,103 @@
...
+  height: 60px;
+  padding: 0;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  transition: color 0.3s;
+}
+
+.header__nav__item-4 .header__nav__link:hover {
```
Comment (jv-aquino) [inline src/style.css:85]: Every link should have this property

**Review by jv-aquino — CHANGES_REQUESTED** [review]
You need to change some things in the implementations to follow the best practices, but your code organization is good

File: src/index.html (hunk 1 above)
Comment (eduvinagre) [reply src/index.html:45]: Hi João, I'm having a hard time understanding this. What would be the right way? 
```
<nav class="header nav">
  <ul class="header nav list">
``` 
Would that be the right way of doindo this?

Comment (eduvinagre) [reply src/index.html]: nesting fixed

File: src/style.css (hunk 3 above)
Comment (eduvinagre) [reply src/style.css:37]: fixed

File: src/index.html (hunk 2 above)
Comment (eduvinagre) [reply src/index.html]: fixed

File: src/style.css (hunk 4 above)
Comment (eduvinagre) [reply src/style.css]: fixed

**Pull request closed without merging**